packages/bridge/src/vl/legends/index.ts

```diff
@@ -136,6 +136,7 @@ interface LegendOptions {
   onLoad?: () => void;
   // Should the legend repaint after layer updates or just initially
   dynamic?: boolean;
+  config?: (othersLabel?: string, samples?: number) => object;
```

Comment:
I think the typing should be:

```suggestion
  config?: { othersLabel?: string, samples?: number };
```

As it is, it means a function that takes those 2 arguments and returns an object, doesn't it?

Reply:
ups, yes, you're right!